source/getting-started/concepts-terminology.markdown

```diff
@@ -0,0 +1,43 @@
+---
+title: "Concepts and terminology"
+description: "User explanation of Home Assistant basics"
+---
+Now you're in Home Assistant, let's go over important parts of Home Assistant and what they do.
+
+### Dashboards
+Customizable pages to display information connected to and available in Home Assistant. Dashboards are the first thing you see after the [onboarding process](/getting-started/onboarding). More details can be found [here](/dashboards).
+
+![Dashboard](/images/getting-started/lovelace.png)
+### Integrations
+Software that allows Home Assistant to connect to other software and platforms. For example, Philips Hue integration allow Home Assistant to talk to the Hue Bridge and read data such as the state of your lights through to controlling the lights themselves.
+
+![Integrations](/images/getting-started/integrations.png)
+
+A full list of compatible integrations can be found [here](/integrations).
+
+Once an integration has been added, the hardware and/or data are represented in Home Assistant as [devices and entities](#devices--entities).
+
+### Devices & Entities
+Devices are logical grouping for one or more entities. A device may represent a physical device and the device may have one or more sensors which appear as entities associated to the device. For example, a motion sensor is represented as a device and the motion may provide motion detection, temperature and light levels as entities. Entities have states such as `on` when motion is detected and `off` where is no motion.
```

Comment:
```suggestion
Devices are a logical grouping for one or more entities. A device may represent a physical device and the device may have one or more sensors which appear as entities associated to the device. For example, a motion sensor is represented as a device and may provide motion detection, temperature, and light levels as entities. Entities have states such as `on` when motion is detected and `off` when there is no motion.
```

Reply:
I would also suggest no to use code-fence `on` and `off` in this case. And just keep it with normal text for this part.